subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -2509,3 +2512,13 @@ static void disabled_cb(void *param)
 {
 	k_sem_give(param);
 }
+
+void ull_set_done_extra_type(uint8_t type)
```

Comment:
```suggestion
struct event_done_extra *ull_event_done_extra_type_set(uint8_t type)
```

Returning the `struct event_done_extra *` will be useful when other places can reuse this function and then update other fields in the done_extra as necessary.